macros/Compat.ejs

```diff
@@ -264,92 +268,84 @@ function getSupportClass(supportInfo) {
   return cssClass;
 }
 
-/*
-Generate the note for a browser flag or preference
-First checks version_added and version_removed to create a string indicating when
-a preference setting is present. Then creates a (browser specific) string
-for either a preference flag or a compile flag.
-
-// TODO Need to localize this
-
-`supportData` is a support_statement
-`browserId` is a compat_block browser ID
-*/
-function writeFlagsNote(supportData, browserId) {
-  let output = '';
-
-  const firefoxPrefs = ' To change preferences in Firefox, visit about:config.';
-  const chromePrefs = ' To change preferences in Chrome, visit chrome://flags.';
-
-  if (typeof(supportData.version_added) === 'string') {
-    output = `From version ${supportData.version_added}`;
+/**
+ * Generate the note for a browser flag or preference
+ * First checks version_added and version_removed to create a string indicating when
+ * a preference setting is present. Then creates a (browser specific) string
+ * for either a preference flag or a compile flag.
+ *
+ * @param {object} supportData is a support_statement
+ * @param {string} browserId is a compat_block browser ID
+ *
+ * @return {string} The note for the flags information.
+ */
+function writeFlagsNote (supportData, browserId) {
+  // TODO: Move these to the mdn-browser-compat-data package.
+  const firefoxPrefs = 'about:config';
+  const chromePrefs = 'chrome://flags';
```

Comment:
See&nbsp;https://github.com/mdn/browser-compat-data/pull/3407 for&nbsp;unhard&#x2011;coding&nbsp;this.